root/search/components/WorkResults.js

```diff
@@ -58,7 +58,7 @@ const WorkResults = ({
       query={query}
       results={results}
     />
-    {$c.user && !$c.user.is_editing_disabled ? (
+    {$c && $c.user && !$c.user.is_editing_disabled ? (
```

Comment:
That's a bit unfortunate since we know it always exists here. Can you add `+$c: CatalystContextT,` within the definition for `ResultsPropsT` after the `...InlineResultsPropsT<T>,` line to indicate it's always defined in `ResultsPropsT`?

Reply:
It's not though - it's for example not defined for InstrumentResults, since that doesn't load $c.

Reply:
Another option would be to have ResultsPropsT and ResultsPropsWithContextT and use the appropriate type each time.

Reply:
Changed to use ResultsPropsWithContextT